### PR TITLE
Fix https://github.com/webksde/ddev-vscode-devcontainer-drupal-templa…

### PR DESCRIPTION
**Pull request by JPustkuchen**
Fix https://github.com/webksde/ddev-vscode-devcontainer-drupal-template/issues/140

### CODE REVIEW

File: .ddev/commands/host/drowl-init-from-existing.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ if [ $# -eq 0 ]; then
     y | Y | yes | Yes | YES)
       echo -e $'\e\n[32mOk! requiring development tools...\n\e[0m'
       ddev composer require --dev cweagans/composer-patches szeidler/composer-patches-cli drupal/admin_toolbar drupal/backup_migrate drupal/examples drupal/stage_file_proxy drupal/devel drupal/devel_debug_log drupal/devel_php drupal/coder drupal/examples
-      ddev composer require --dev drupal/core-dev:^10 drush/drush phpunit/phpunit:^9.5 phpspec/prophecy-phpunit phpstan/phpstan mglaman/phpstan-drupal phpstan/phpstan-deprecation-rules phpstan/phpstan-phpunit phpstan/extension-installer -W
```
Comment (joshsedl) [inline .ddev/commands/host/drowl-init-from-existing.sh:53]: This seems unrelated? There probably was a reason to have an explicit version set here.

File: .ddev/commands/host/drowl-init.sh [hunk 2]
```diff
@@ -54,7 +54,7 @@ ddev composer require composer/installers cweagans/composer-patches szeidler/com
 
 # Add DEV dependencies (but no modules due to their database relationship):
 ddev composer require --dev drupal/core-dev:^${DRUPAL_VERSION}
-ddev composer require --dev drush/drush phpunit/phpunit:^9.5 drupal/coder phpspec/prophecy-phpunit phpstan/phpstan mglaman/phpstan-drupal phpstan/phpstan-deprecation-rules phpstan/extension-installer phpstan/phpstan-phpunit kint-php/kint
```
Comment (joshsedl) [inline .ddev/commands/host/drowl-init.sh:57]: Same here as above.

**Comment by joshsedl** [comment]
Furthermore, the "phpunit-coverage" script should be rewritten.

**Comment by JPustkuchen** [comment]
@joshsedl in my tests the version constraints were not needed anymore. You're right we should be careful with this, but if we can't find a reason anymore, let's remove it.

**Comment by joshsedl** [comment]
This needs further adjustments, the code coverage command is not properly working. We might also fix the "Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port)." problem here, while we are at it.

**Comment by joshsedl** [comment]
Alright, all done now everything works great again!
The "Could not connect to debugging client" warning couldn't get resolved. But it was there before, so very unrelated. It also doesn't break anything (just looks like it). I'll create a follow-up issue for that. RTBC!

**Comment by joshsedl** [comment]
Created the seperate issue here: https://github.com/webksde/ddev-vscode-devcontainer-drupal-template/issues/143
